routes/hotels.js

```diff
@@ -0,0 +1,64 @@
+const mongoose = require('mongoose')
+const express = require('express')
+const { Hotel } = require('../models/hotel')
+const router = express.Router()
+
+router.get('/', (req, res) => {
+  Hotel.find()
```

Comment:
Can you make it with await? It looks much better without using then. Please watch first link with JavaScript that should be in pd-przydatne-info.